site/app/controllers/OfficeHoursQueueController.php

```diff
@@ -59,7 +59,10 @@ public function openQueue() {
         if ($token !== null) {
             preg_match_all($re, $token, $matches_token, PREG_SET_ORDER, 0);
         }
-        if (count($matches_code) !== 1 || count($matches_token) !== 1) {
+        if (
+            is_array($matches_code) && count($matches_code) !== 1
+            || is_array($matches_token) && count($matches_token) !== 1
+        ) {
```

Comment:
We could probably just initialize these variables to empty arrays (before the preg_match_all call) and then we can always check the count without issue.

Reply:
Good idea. Fixed!